google_shopping.js

```diff
@@ -1,5 +1,54 @@
 var data = require("./products.json")
 
 // Write your solutions below
+var q = 0
+for (i = 0; i < data["items"].length; i++){
+
+	if (data["items"][i].product.inventories[0].availability === "inStock"){
+		q++;
+	}
+	
+}
+console.log("There are " + q + " items In Stock.");
+
+var q = 0;
+for (i = 0; i < data["items"].length; i++){
+	if (data["items"][i].product.images.length > 1){
+		q++;		
+	}
+}
+console.log("There are " + q + " items with more than one image link.");
+
+var q = 0;
+for (i = 0; i < data["items"].length; i++){
+	if (data["items"][i].product.brand.includes("anon")){
```

Comment:
Ops! There's a typo here. Includes is a great method to use, but it isn't supported across all browsers. An alternative method would be to use .indexOf or the equality operator.

```if (data["items"][i].product.brand.toLowerCase() === ("canon")){```
```if (data["items"][i].product.brand.indexOf("canon") !== -1){```